apps/ui/src/components/Modal/Vote.vue

```diff
@@ -29,9 +29,13 @@ const {
   fetch: fetchVotingPower,
   reset: resetVotingPower
 } = useVotingPower();
+const { votes } = useAccount();
 
 const loading = ref(false);
-const form = ref<Record<string, string>>({ reason: '' });
+const form = ref<Record<string, string>>({
+  reason:
+    votes.value[`${props.proposal.network}:${props.proposal.id}`]?.reason || ''
+});
```

Comment:
Maybe an edge case, but i found a small bug

- Create two proposals and vote with different reason on both
- Try to edit vote on first proposal, keep the modal open
- Change the URL to the second proposal
- Then it will show the reason of first proposal

Reply:
This seems to affect the whole modal data, and not only the reason.

- Choice
- Voting power
- Reason
- Even the voting modal itself, which should not be available if the proposal is not active

Voting power seems to refresh automatically on user change though, thanks to the new VotingPower composable

Reply:
Will open another PR to fix these

Reply:
Will be fixed in https://github.com/snapshot-labs/sx-monorepo/pull/612, no changes needed in this one